packages/h/handbrake/package.yml

```diff
@@ -18,6 +18,7 @@ builddeps  :
     - pkgconfig(ffnvcodec)
     - pkgconfig(gstreamer-plugins-base-1.0)
     - pkgconfig(gtk+-3.0)
+    - pkgconfig(gtk4-wayland)
```

Comment:
Since they migrated the UI to GTK4, does this need GTK 3 at all? There are also a bunch of libraries that it no longer links against, are any of them in the list that can be removed?

The `pkgconfig` for GTK4 can also just be `pkgconfig(gtk4)`.